Latex/vorlage.tex

```diff
@@ -314,4 +316,4 @@
   \caption{#3}
   \label{fig:#4}
   \end{figure}}
-\newcommand{\fn}[1]{\footnote{\hspace{0.5em}#1}}
\ No newline at end of file
+\newcommand{\fn}[1]{\footnote{\hspace{0.5em}#1}}
```

Comment:
was ist hier schon wieder passiert? :D

Reply:
ich habe exakt keine ahnung, is ja aber auch die selbe zeile --> ignorieren